eventuate-local-java-cdc-connector-mysql-binlog/src/test/java/io/eventuate/local/mysql/binlog/MySqlBinaryLogClientForPerformanceTesting.java

```diff
@@ -0,0 +1,266 @@
+package io.eventuate.local.mysql.binlog;
+
+
+import com.github.shyiko.mysql.binlog.BinaryLogClient;
+import com.github.shyiko.mysql.binlog.event.*;
+import com.github.shyiko.mysql.binlog.event.deserialization.EventDeserializer;
+import com.github.shyiko.mysql.binlog.event.deserialization.NullEventDataDeserializer;
+import com.google.common.collect.ImmutableSet;
+import io.eventuate.common.eventuate.local.BinLogEvent;
+import io.eventuate.common.eventuate.local.BinlogFileOffset;
+import io.eventuate.common.jdbc.EventuateSchema;
+import io.eventuate.local.common.*;
+import io.eventuate.local.db.log.common.DbLogMetrics;
+import org.mockito.Mockito;
+
+import javax.sql.DataSource;
+import java.util.*;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+public class MySqlBinaryLogClientForPerformanceTesting {
```

Comment:
Can this be replaced with the real `MySqlBinaryLogClient` with appropriate mocks for dependencies?

Reply:
I will check.